community/ndp/runtime/src/test/java/org/neo4j/ndp/runtime/integration/RecordingCallback.java

```diff
@@ -152,14 +152,14 @@ public boolean isSuccess()
 
     public static class Result extends Success
     {
-        private final Object[] streamValues;
+        private final Record[] streamValues;
```

Comment:
I would wait for someone else to confirm this change `Object -> Record`. Besides this, this pr is good to me